### PR TITLE
Finished the ✨ welcome page ✨ 

### PR DESCRIPTION
**Pull request by bejanaf**
Dear Web Devs, I finished my ✨  welcome page ✨ and some little changes / bugs fixes in my other pages, like:
- changed the get route in my server.js and App.js to make the address unique
- changed the Route in my App.js
- added the navigation to News.js, Bookmark.js and Wallet.js
- added some default colors to my GlobalStyles.js
Would be thankful for a professional look :)

### CODE REVIEW

File: client/src/GloabalStyle.js [hunk 1]
```diff
@@ -28,6 +29,7 @@ export default createGlobalStyle`
 
 body, h1, h2, h3, h4, h5, h6, ol, ul {
     margin: 0;
+    margin-top: 0.5rem;
```
Comment (heleah) [inline client/src/GloabalStyle.js:32]: the file should be spelled "GlobalStyle.js" (not Gloabal) (: ✨

File: client/src/components/Navigation.js [hunk 2]
```diff
@@ -7,56 +7,78 @@ import { ReactComponent as WalletIcon } from '../image/wallet.svg';
...
-    height: 2rem;
-    fill: white;
-  }
-`;
-
 const NavigationWrapper = styled.div`
-  display: flex;
-  justify-content: center;
-`;
+  background: repeating-linear-gradient(
```
Comment (heleah) [inline client/src/components/Navigation.js:36]: please consider sorting the css styles in alphabetical order (:

File: client/src/components/Navigation.js [hunk 3]
```diff
@@ -7,56 +7,78 @@ import { ReactComponent as WalletIcon } from '../image/wallet.svg';
...
+    linear-gradient(to top, var(--black) 60%, var(--primary));
 
-const NavBar = styled.div`
-  border: 3px solid black;
-  border-radius: 100vh;
+  align-items: center;
+  bottom: 0;
+  box-shadow: 5px 5px 5px var(--primary);
   display: flex;
+  color: white;
```
Comment (heleah) [inline client/src/components/Navigation.js]: please consider sorting the css styles in alphabetical order (:

File: client/src/pages/WelcomePage.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  display: flex;
+  flex-direction: column;
+
+  svg {
+    filter: drop-shadow(0rem 0rem 1rem var(--primary));
+    margin-top: 2.5rem;
+  }
+`;
+
+const HeadLiner = styled.h1``;
```
Comment (heleah) [inline client/src/pages/WelcomePage.js]: if you do not apply any styles to a styled component, you do not need it (: ✨  you can just use ```<h1></h1> ``` instead

**Review by heleah — APPROVED** [review]
✨  Looks alright to me! (: :D (; ✨

File: client/src/components/Navigation.js (hunk 2 above)
Comment (bejanaf) [reply client/src/components/Navigation.js:36]: fixed :)

File: client/src/components/Navigation.js (hunk 3 above)
Comment (bejanaf) [reply client/src/components/Navigation.js]: fixed :)

File: client/src/pages/WelcomePage.js (hunk 4 above)
Comment (bejanaf) [reply client/src/pages/WelcomePage.js]: fixed :)

**Comment by bejanaf** [comment]
Thank you very much for your review. ✨  I fixed them all :)

**Pull request merged**